addons/web_editor/static/lib/odoo-editor/src/utils/sanitize.js

```diff
@@ -187,8 +187,11 @@ class Sanitize {
 
             // Ensure elements which should not contain any content are tagged
             // contenteditable=false to avoid any hiccup.
+            // TODO: improve the way we permit the edition of icons that are in
+            // the block social media.
             if (
-                (isMediaElement(node) || node.tagName === 'HR') &&
+                ((isMediaElement(node) && !node.closest('.s_social_media')) ||
+                node.tagName === 'HR') &&
```

Comment:
Not a fan of `_getContentEditableAreas` & cie but since we use it to fix this bug in stable, can't you it here to make a generic fix instead of using a *specific* *website* class in a *web_editor* *lib* file handling *generic* things? I am sure there is a way here, somehow, to say "add contenteditable false on those elements... unless they are explicitely marked as "content editable areas" by `getContentEditableAreas`.